target/linux/lantiq/files/arch/mips/boot/dts/lantiq/ar9_avm_fritz7312.dts

```diff
@@ -183,7 +183,7 @@
 	req-mask = <0xf>;
 	reset-gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
 
-	wifi@0,0 {
+	wifi@e,0 {
```

Comment:
How does the kernel come up with `e,0`? Where is the `e` coming from?

Reply:
Thank you for your explanation. In short:
```
reg = <0x7000 0 0 0 0>
dev = printf("%x", (reg[0] & 0xf800) >> 11)
func = printf("%x", (reg[0] & 0x700) >> 8)
node_name = "xxx@$dev,$func"
```
Ref: https://github.com/torvalds/linux/blob/0f70f5b08a47a3bc1a252e5f451a137cde7c98ce/scripts/dtc/checks.c#L923